src/Spd.Manager.Licence/LicenceAppManagerBase.cs

```diff
@@ -51,17 +51,26 @@ protected async Task<decimal> CommitApplicationAsync(LicenceAppBase licAppBase,
             HasValidSwl90DayLicence = HasSwl90DayLicence
         }, ct);
         LicenceFeeResp? licenceFee = price?.LicenceFees.FirstOrDefault();
+
+        bool isNewOrRenewal = licAppBase.ApplicationTypeCode == Shared.ApplicationTypeCode.New || licAppBase.ApplicationTypeCode == Shared.ApplicationTypeCode.Renewal;
+        ApplicationStatusEnum status;
+
         if (licenceFee == null || licenceFee.Amount == 0)
         {
-            if (companionAppId != null) await _licAppRepository.CommitLicenceApplicationAsync((Guid)companionAppId, ApplicationStatusEnum.Submitted, null, ct);
-            await _licAppRepository.CommitLicenceApplicationAsync(licenceAppId, ApplicationStatusEnum.Submitted, null, ct);
+            status = isNewOrRenewal && !IsAuthenticated ? ApplicationStatusEnum.ApplicantVerification : ApplicationStatusEnum.Submitted;
         }
         else
         {
-            //companionAppId is the swl for sole proprietor which the business would pay for it, therefore the licence fee should be null here.
-            if (companionAppId != null) await _licAppRepository.CommitLicenceApplicationAsync((Guid)companionAppId, ApplicationStatusEnum.PaymentPending, null, ct);
-            await _licAppRepository.CommitLicenceApplicationAsync(licenceAppId, ApplicationStatusEnum.PaymentPending, licenceFee.Amount, ct);
+            status = isNewOrRenewal && !IsAuthenticated ? ApplicationStatusEnum.ApplicantVerification : ApplicationStatusEnum.PaymentPending;

```

Comment:
even thought it is new and it is not authenticated, this situation should still be payment pending. Dynamics will check its status when user paid.

Reply:
changed to just be Payment Pending